specifyweb/frontend/js_src/lib/components/FormMeta/CarryForward.tsx

```diff
@@ -367,8 +407,18 @@ function CarryForwardCategory({
                 }
               >
                 <Input.Checkbox
-                  checked={f.includes(carryForward, field.name)}
-                  disabled={isUnique}
+                  checked={
+                    f.includes(carryForward, field.name) ||
+                    (isBulkConfig === true &&
+                      (field.localization.isrequired ||
+                        field.overrides.isRequired))
```

Comment:
could you extract this into a variable to not duplicate it twice? it's a pretty complex expression